stubs/Pygments/@tests/requirements-stubtest.txt

```diff
@@ -0,0 +1,2 @@
+docutils
```

Comment:
This shouldn't be required, since the types already depend on `types-docutils`. This means that the implementation isn't used anyway.